src/main/java/co/cask/gcp/bigquery/BigQuerySource.java

```diff
@@ -112,8 +112,8 @@ public void prepareRun(BatchSourceContext context) throws Exception {
     job.setOutputKeyClass(LongWritable.class);
     job.setOutputKeyClass(Text.class);
 
-    setInputFormat(context);
     emitLineage(context);
+    setInputFormat(context);
```

Comment:
Let's put a comment about the positioning so that later we can avoid this in refactoring.